build-system/tasks/check-sourcemaps.js

```diff
@@ -33,6 +33,7 @@ const sourcemapUrlMatcher =
 // Mapping related constants
 const expectedFirstLineFile = 'src/polyfills/array-includes.js';
 const expectedFirstLineCode = 'function includes(value, opt_fromIndex) {';
+const expectedColStart = "'use strict;'".length; // Code begins after an initial 'use strict'.
```

Comment:
Now that you're checking for the column, can you update the JSDoc for `checkSourceMappings()` (line 126) to the following?

```js
5. Check if the filename, line of code, and column match expected sentinel values.
```

Reply:
We were already checking column, happy to add this comment though